leveldb-server/src/db.ts

```diff
@@ -104,50 +99,63 @@ export class DB extends EventEmitter {
         patches: patches.slice(-NUM_PATCHES_TO_KEEP).concat(patch),
         updatedAt: hrnano(),
       }));
-      this.emit('patch', key, patch);
+      // Trigger return from outstanding `poll`.
+      this.emitter.emit('patch', key, patch);
     }
   }
 
   /**
    * Gets a single document.
    * @return - value or undefined if key doesn’t exist.
    */
-  public async get<T extends Serializable = any>(key: string): Promise<T | undefined> {
-    const versioned = await this.getWithMeta(key);
+  public async get(ctx: ServerOnlyContext, key: string): Promise<Serializable | undefined> {
+    const versioned = await this.getWithMeta(ctx, key);
     if (versioned === undefined) {
       return undefined;
     }
-    return versioned.value;
+    return (versioned as any).value;
```

Comment:
`versioned` might be a `Tombstone`.  In that case it has no `value` field but I still want to get `undefined`.  Tombstone cannot have a `value: undefined`, because that isn't JSONable, and we'll _anyway_ get it with no `value` field.